golem_cli/src/status.rs

```diff
@@ -69,13 +69,15 @@ pub async fn run() -> Result</*exit code*/ i32> {
             table.add_empty_row();
             table.add_row(row!["address", &id.node_id]);
             table.add_row(row!["amount", format!("{} NGNT", &payment_status.amount)]);
+            table.add_empty_row();
             table.add_row(row![
                 "pending",
-                format!("{} NGNT", &payment_status.incoming.total_pending())
+                format!("{} NGNT", &payment_status.incoming.total_pending().0)
```

Comment:
We could also show the number of total pending payments, similar to unconfirmed below

Reply:
fixed
```
$ golemsp settings set --cpu-per-hour 5
$ golemsp settings show
node name: "reqc1"
shared resources:
	cores:	2
	memory:	1 GiB
	disk:	10 GiB


Pricing:

	 0.01 NGNT for start
	  0.2 NGNT per hour
	    5 NGNT per cpu hour
```